CITATION.cff

```diff
@@ -0,0 +1,23 @@
+# Parser settings.
+cff-version: 1.2.0
+message: Please cite this crate using these information.
+
+# Version information.
+date-released: 2022-11-12
+version: 0.4.23
+
+# Project information.
+abstract: Date and time library for Rust
+authors:
+  - alias: .
+    # email:
+    family-names: .
+    given-names: .
+    # website:
```

Comment:
The authors section is a place to present oneself.  Thus, and since the format originates from a rather academic audience, there are plenty of *optional* fields to fill out:  website, email address, postal address (mostly for companies), scientific researcher ID, ...

I added `email` and `website` as comments in order to present some examples of rather common further information.  Any author can freely decide how they would like to be named, *if* they would like to be named at all.